lib/modules/manager/terraform/extract.spec.ts

```diff
@@ -152,6 +152,34 @@ describe('modules/manager/terraform/extract', () => {
           currentValue: 'v0.1.0',
           datasource: 'github-tags',
         },
+        {
+          packageName: 'git@git.example.com:group/repo/foo',
+          currentValue: 'v1.0.0',
+          depType: 'module',
+          depName: 'git.example.com:group/repo/foo',
+          datasource: 'git-tags',
```

Comment:
Noting here that these are clearly git refs, not tags. Most of the time they're tags, but sometimes they could be branches. It's probably a flaw that we're using git-tags instead of git-refs are the datasource, but that doesn't seem to be a flaw you added?